bindings/rust/wasmedge-sdk/examples/run_func_in_aot_mode.rs

```diff
@@ -0,0 +1,37 @@
+use wasmedge_sdk::{
+    config::{
+        CommonConfigOptions, CompilerConfigOptions, ConfigBuilder, HostRegistrationConfigOptions,
+    },
+    params, Compiler, CompilerOutputFormat, Vm, WasmVal,
+};
+
+#[cfg_attr(test, test)]
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // create a Config context
+    let config = ConfigBuilder::new(CommonConfigOptions::new().bulk_memory_operations(true))
+        .with_compiler_config(
+            CompilerConfigOptions::new()
+                .interruptible(true)
+                .out_format(CompilerOutputFormat::Native),
```

Comment:
You set the output format to `Native`, however, set the output file name to `.wasm`.

Reply:
Accepted. Thanks!